_sass/design-system/_settings.scss

```diff
@@ -80,7 +80,7 @@ $red:#f44336;
 $red100:#ffcdd2;
 $red500:#f44336;
 $red700:#d32f2f;
-$redAccent:#eb4d5c;
+$redAccent:#ec515f;
```

Comment:
No big deal here, but I'm wondering why we switch from numerical naming for the colors to the text name.  Seems like we'd want it to be consistent one way or the other.